From 3c5613570dcbb1cc13303445f9822e3945097d6d Mon Sep 17 00:00:00 2001
From: Matt Gumbel <matthew.k.gumbel@intel.com>
Date: Thu, 7 Nov 2013 13:23:11 -0800
Subject: Treat java-6-oracle as equivalent to java-6-sun

After Oracle bought sun, Java 1.6 install directory was re-branded as
Oracle. Should still be valid for Android builds though.

This JAVA_HOME variable only comes into play during the SDK build which
is why all of our builds were find without this patch.

Category: AOSP Enhancement
Upstream-Candidate: Yes
Issue: ADEV-269
Change-Id: Iacde4c3808ba2bc85d2d7a1ee0854c47df7aad50
Signed-off-by: Matt Gumbel <matthew.k.gumbel@intel.com>
---
 envsetup.sh |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/envsetup.sh b/envsetup.sh
index 0661fe6..42ab76a 100644
--- a/envsetup.sh
+++ b/envsetup.sh
@@ -1303,7 +1303,11 @@ function set_java_home() {
                 export JAVA_HOME=/System/Library/Frameworks/JavaVM.framework/Versions/1.6/Home
                 ;;
             *)
-                export JAVA_HOME=/usr/lib/jvm/java-6-sun
+                if [ -d /usr/lib/jvm/java-6-oracle ]; then
+                    export JAVA_HOME=/usr/lib/jvm/java-6-oracle
+                else
+                    export JAVA_HOME=/usr/lib/jvm/java-6-sun
+                fi
                 ;;
         esac
     fi
-- 
1.7.4.1

